Introduce machine key set flag

Scharfenberg opened the pull request:
<!--
Thank you for contributing to this project!

- This project has a Contributor Code of Conduct: https://voxpupuli.org/coc/
- Please check that here is no existing issue or PR that addresses your problem.
- Our vulnerabilities reporting process is at https://voxpupuli.org/security/

-->
#### Pull Request (PR) description
I've run into the problem that private keys imported from .pfx files are not accessible when using the certificates within IIS. Obviously this is because the keys are installed into the user key storage instead of the machine key storage.

This was fixed by adding the 'MachineKeySet' storage flag when calling X509Certificates.import() (also refer to: https://stackoverflow.com/questions/36867243/what-is-the-impact-of-the-persistkeyset-storageflag-when-importing-a-certifica). This flag is controlled by the new 'use_machine_key_storage' parameter.

Additional change:
 - removed the 'Remove-Item $MyINvocation.InvocationName' in both Powershell template files as these are only useful if temporary ps1 files are created. This is not the case any more.

#### This Pull Request (PR) fixes the following issues
Addresses #25 
@hpAB also suggests to introduce this flag within the comments

bastelfreak commented:
Hey @Scharfenberg, thanks for the PR. Can you please rebase against our latest master branch?

Scharfenberg commented:
Finally I've rebased my code - I was reminded by the Vox Pupuli GitHub Bot to do so. I've noticed that both changes I've introduced in my version are already included in the master branch meanwhile. So this pull request is obsolete,
